src/components/TimePickerInput/index.jsx

```diff
@@ -116,7 +132,11 @@ TimePickerInput.propTypes = {
   /**
    * To specify the values to be displayed inside the TimePicker.
    */
-  value: PropTypes.string,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  /**
+   * To specify the values to be displayed inside the TimePicker.
+   */
+  defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
Add `defaultValue`, `clearIcon `, `disabled ` and `type` in [TimePickerInputProps](https://github.com/bigbinary/neeto-ui/blob/ed06c57be2a18497c468119a6be9b4f9fb5545ec/index.d.ts#L728).